lib/kino/remote_cell.ex

```diff
@@ -0,0 +1,88 @@
+defmodule Kino.RemoteCell do
+  @moduledoc false
+
+  use Kino.JS, assets_path: "lib/assets/remote_cell"
+  use Kino.JS.Live
+  use Kino.SmartCell, name: "Remote cell"
+
+  @default_code ":ok"
+
+  @impl true
+  def init(attrs, ctx) do
+    fields = %{
+      "assign_to" => attrs["assign_to"] || "",
+      "cookie" => attrs["cookie"] || "",
+      "node" => attrs["node"] || ""
+    }
+
+    ctx = assign(ctx, fields: fields)
+
+    {:ok, ctx, editor: [attribute: "code", language: "elixir", default_source: @default_code]}
+  end
+
+  @impl true
+  def handle_connect(ctx) do
+    payload = %{fields: ctx.assigns.fields}
+    {:ok, payload, ctx}
+  end
+
+  @impl true
+  def handle_event("update_field", %{"field" => field, "value" => value}, ctx) do
+    ctx = update(ctx, :fields, &Map.put(&1, field, value))
+    broadcast_event(ctx, "update_field", %{"fields" => %{field => value}})
+
+    {:noreply, ctx}
+  end
+
+  @impl true
+  def to_attrs(ctx) do
+    ctx.assigns.fields
+  end
+
+  @impl true
+  def to_source(%{"node" => ""}), do: ""
+  def to_source(%{"cookie" => ""}), do: ""
+  def to_source(%{"code" => ""}), do: ""
+
+  def to_source(%{"code" => code} = attrs) do
+    code = Code.string_to_quoted(code)
+    to_quoted(attrs, code)
+  end
+
+  defp to_quoted(%{"node" => node, "cookie" => cookie, "assign_to" => var}, {:ok, code}) do
+    var = if Kino.SmartCell.valid_variable_name?(var), do: var
+    call = build_call(code) |> build_var(var)
+
+    quote do
+      node = unquote(String.to_atom(node))
+      cookie = unquote(String.to_atom(cookie))
+      Node.set_cookie(node, cookie)
+      Node.connect(node)
```

Comment:
I think we actually don't need this call. `:erpc` connects automatically! Please verify:

```suggestion
```